aziot/src/main.rs

```diff
@@ -7,48 +7,43 @@
     clippy::let_unit_value,
     clippy::similar_names,
     clippy::too_many_lines,
-    clippy::type_complexity
+    clippy::type_complexity,
+    clippy::clippy::module_name_repetitions
```

Comment:
`s/clippy::clippy::/clippy::/`

And sort it.

Reply:
Huh, I'm more surprised that `clippy::clippy::foo` not only validates, but has the same behavior as `clippy::foo`. That's weird

Reply:
>And sort it.

Reply:
huh, I kinda assumed `rustfmt` would sort it automatically, but no, it doesn't. I now see why you're annoyed by the fact that there's no trailing comma - spending 5 seconds fixing that build error is time I'll never get back lol